### PR TITLE
Validate WHY

### PR DESCRIPTION
**Pull request by Wenwifhat**
# Validate [WHY](https://solscan.io/token/WHYoaBumcmxCqw38y2mjs4cVkCBgwiDizbMVvcejmGT)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/wenwifhat_sol/status/1809129629912850888?s=46
- Coingecko URL: https://www.coingecko.com/en/coins/wenwifhat
- CMC URL:https://coinmarketcap.com/currencies/wenwifhat/

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [x] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Review by Dota15 — COMMENTED** [review]
Wenwifhat will take over crypto !!!!!!!!!!!!!!

**Comment by justmeLFG** [comment]
great project with real utilities. long term. very rare to find on sol.

**Review by YesheadNL — COMMENTED** [review]
I approve this. Project is legit and dev is very serious and based

**Comment by alldldup** [comment]
I approve of this dev. He is consistently adding value to the space and taking part in legitimate projects. He is committed to this path and should be held in that regard.

**Review by lglass1967 — COMMENTED** [review]
This is looking bullish!!!!! LFG!!!!!💯

**Comment by lglass1967** [comment]
> # Validate [WHY](https://solscan.io/token/WHYoaBumcmxCqw38y2mjs4cVkCBgwiDizbMVvcejmGT)
> ## Attestations (Please provide links):
> * Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://x.com/wenwifhat_sol/status/1809129629912850888?s=46
> * Coingecko URL: https://www.coingecko.com/en/coins/wenwifhat
> * CMC URL:https://coinmarketcap.com/currencies/wenwifhat/
> 
> ## Validation (Please check off boxes):
> * [x]  The metadata provided in the PR matches what is on-chain (Mandatory)
> * [x]  Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
> * [x]  Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)
> 
> ## Acknowledgement (Please check off boxes)
> * [x]  My change matches the format in the file (no spaces between fields).
> * [x]  My token is already live and trading on Jupiter.
> * [x]  !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
> * [x]  Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

**Comment by lglass1967** [comment]
Really Looking forward to the future of this. LFG!!!!💯

**Review by IanHicky — COMMENTED** [review]
Sus.

**Comment by IanHicky** [comment]
all WHY launches looking like rugg. farmer project?

**Comment by Dannyboy241** [comment]
> all WHY launches looking like rugg. farmer project?

And all have individual teams ? Have u seen pinksale ?

**Comment by Dannyboy241** [comment]
> all WHY launches looking like rugg. farmer project?

Also our job is to facilitate presales we do that we can’t help the overall market conditions and the teams that launch via wenpad look at solex and pinksale too

**Review by lglass1967 — COMMENTED** [review]
Great community, great Dev. This isn’t something that will disappear but a project that will thrive.

**Comment by Dannyboy241** [comment]
> all WHY launches looking like rugg. farmer project?

And this review isn’t for our utility it’s for our token which even in the market conditions is holding stronger than every other project out there

**Comment by Pkabutu** [comment]
$Why is the best,few launches and filled very fast on their launch pad.
Don't fade based Dev

**Review by Jymfletzfur — APPROVED** [review]

**Comment by meeche03** [comment]
one of the best teams in defi crypto. Love what they are building here

**Comment by imcryptonaire** [comment]
We are committed to building sustainability and legitimacy on a 0 tax network . Bringing fundamental
Values of trust for our holders. Almost 5 months old itself is an achievement and one of the most stable charts on solana, not only that security wise we are verified on phantom and have been recently added to blowfish allow list

File: validated-tokens.csv
Comment (imcryptonaire) [inline validated-tokens.csv:1]: We are committed to building sustainability and legitimacy on a 0 tax network . Bringing fundamental
Values of trust for our holders. Almost 5 months old itself is an achievement and one of the most stable charts on solana, not only that security wise we are verified on phantom and have been recently added to blowfish allow list

File: validated-tokens.csv [hunk 1]
```diff
@@ -1863,3 +1863,4 @@ BRETT GOLD,BRETTGOLD,F6ExBzKdLRcJkCAknQgfbhRXX78EhqoNxPnegJWPpump,6,https://stat
 Purple Pepe,PURPE,HBoNJ5v8g71s2boRivrHnfSB5MVPLDHHyVjruPfhGkvL,1,https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f617277656176652e6e65742f574575676b676c4243354c654138544175764f763878325f3777534744554e324938315f6e597032422d41,true
 21X,21X,6r4PCVaX4rYN9WdbXwVWAQL4djFoUaeBMsq8Cxc6NApZ,9,https://statics.solscan.io/cdn/imgs/s60?ref=68747470733a2f2f6261666b7265696671643577616536356f337a6c6b687772756e33726a6a6772656c6f6e356836343277727364736379636c75686770757a3771792e697066732e6e667473746f726167652e6c696e6b2f,true 
 BAKED,BAKED,CQbXk942c6GPcRwtZ2WMFP5JcQ9NqbXtb8jUewBi7GoT,6,https://raw.githubusercontent.com/Baked-Resources/solana-token/main/baked_logo.jpg,true
+WENWIFHAT,WHY,WHYoaBumcmxCqw38y2mjs4cVkCBgwiDizbMVvcejmGT,9,https://zusqmjtp6ujez6gm5rxjzouetdesm2yx2rotyroljao25ri4ntda.arweave.net/zSUGJm_1Ekz4zOxunLqEmMkmaxfUXTxFy0gdrsUcbMY,true
```
Comment (Pkabutu) [inline validated-tokens.csv:1866]: Great project,$Why to the moon

**Review by Pkabutu — COMMENTED** [review]
Great launches happened already on Wenpad

**Review by imcryptonaire — COMMENTED** [review]
We are committed to building sustainability and legitimacy on a 0 tax network . Bringing fundamental
Values of trust for our holders. Almost 5 months old itself is an achievement and one of the most stable charts on solana, not only that security wise we are verified on phantom and have been recently added to blowfish allow list

**Review by Dota15 — COMMENTED** [review]
We are committed to building sustainability and legitimacy on a 0 tax network . Bringing fundamental
Values of trust for our holders. Almost 5 months old itself is an achievement and one of the most stable charts on solana, not only that security wise we are verified on phantom and have been recently added to blowfish allow list

**Comment by nizamm** [comment]
all information provided is accurate

**Review by nizamm — COMMENTED** [review]
provided information is accurate

File: validated-tokens.csv (hunk 1 above)
Comment (nizamm) [reply validated-tokens.csv:1866]: pull request is accurate

**Review by nizamm — COMMENTED** [review]
approved

**Review by 9yoi — APPROVED** [review]

**Pull request merged**